.github/workflows/conformance.yaml

```diff
@@ -0,0 +1,37 @@
+on:
+  push:
+    branches:
+    - main
+    paths:
+      - 'google-cloud-bigtable/**'
+  pull_request:
+    paths:
+      - 'google-cloud-bigtable/**'
+name: conformance
+jobs:
+  conformance:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby_version: [
+          2.7,
```

Comment:
We're dropping support for Ruby 2.7 shortly. Let's just not add it in the first place for this test.